spec/spec.md

```diff
@@ -623,35 +647,420 @@ the resource. The property ****MUST**** be a unique identifier, such as a [UUID]
         }
       }
       ```
-- `submission_requirements` - The resource ****MAY**** contain this property,
-  and if present, its value ****MUST**** conform to the Submission Requirement
-  Format. If not present, all inputs listed in the `input_descriptors` array are
-  required for submission. The description for the format of this property is in
-  the [`Submission Requirement`](#submission-requirement) section below.
-- `input_descriptors` - The resource ****MUST**** contain this property, and
-  its value ****MUST**** be an array of [[ref:Input Descriptor]] objects. If no
-  `submission_requirements` is present, all inputs listed in the
-  `input_descriptors` array are required for submission. The composition of
-  values under this property are described in the [`Input
-  Descriptors`](#input-descriptors) section below.
+- `submission_requirements` - The [[ref:Presentation Definition]] ****MAY****
+  contain a `submission_requirements` property. If present, its value
+  ****MUST**** be an object conforming to the [[ref:Submission Requirement]]
+  format described in the [`Submission Requirement`](#submission-requirement)
+  section below.
+  
+  If not present, all inputs listed in the `input_descriptors` array are
+  required for submission. 
+
+### Input Descriptors
+[[ref:Input Descriptors]] are objects used to describe the information a
+[[ref:Verifier]] requires of a [[ref:Holder]]. If no
+[[ref: Submission Requirements]] are present, all [[ref:Input Descriptors]]
+****MUST**** be satisfied.
+
+[[ref: Input Descriptor Objects]] contain an identifier, a schema URI that links
+to the schema of the requested input data, and may contain constraints on data
```

Comment:
> schema

When we say "schema" are referring literally to JSONSchema?

Reply:
I do not believe we are. To my understanding we use `schema` values to provide a URI for either a JSONSchema or JSON-LD schema which describes the data in the requested claims. There may even be other schema types I'm not aware of.